src/filter_core/filter_props.c

```diff
@@ -1359,6 +1359,7 @@ GF_BuiltInProperty GF_BuiltInProps [] =
 	{ GF_PROP_PID_CLAMP_DUR, "ClampDur", "Max media duration to process from pid in DASH mode", GF_PROP_DOUBLE, GF_PROP_FLAG_GSF_REM},
 	{ GF_PROP_PID_HLS_PLAYLIST, "HLSPL", "Name of the HLS variant playlist for this media", GF_PROP_STRING, GF_PROP_FLAG_GSF_REM},
 	{ GF_PROP_PID_HLS_GROUPID, "HLSGroup", "Name of HLS Group of a stream", GF_PROP_STRING, GF_PROP_FLAG_GSF_REM},
+	{ GF_PROP_PID_INTRA_ONLY, "IntraOnly", "Indicates the media contains intra frames only (SAP=1)", GF_PROP_BOOL, GF_PROP_FLAG_GSF_REM},
```

Comment:
I think 'GF_PROP_PID_HAS_SYNC' is already signaling this: if present and FALSE, all samples are sync, if present and TRUE, some are sync some are not, if absent, unknown state.
Could you reuse this property instead of defining a new one ?